python/test/t_IndependentCopulaFactory_std.py

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env python
+
+from openturns import *
+
+TESTPREAMBLE()
+
+factory = IndependentCopulaFactory()
+factory.build()
+factory.build(Sample(3, 2))
+factory.build(Point())
```

Comment:
Could we remove the expout file and add an IGNOREOUT flag in the test cmakelist?